lib/installer.js

```diff
@@ -105,6 +106,19 @@ function downloadRelease(version) {
         return yield tc.cacheDir(extPath, "protoc", version);
     });
 }
+function denormalizeVersion(version) {
+    const parsed = semver.parse(version);
+    // We normalized versions like 21.10 into 21.10.0 earlier; to actually get
+    // their file we have to de-normalize them back to 21.10.
+    if (parsed.major > 3) {
+        // We intentionally re-add the `v` prefix here, since the version
+        // coming in is from computeVersion, which also explicitly adds it.
+        version = `v${parsed.major}.${parsed.minor}`;
```

Comment:
I'm not sure if `parsed` has `major` or `minor` fields, it might just be a string:
https://github.com/arduino/setup-protoc/blob/a97892a429d98fae78d26f40334ab7eb616d08b9/node_modules/semver/semver.js#L373